code/modules/paperwork/records/lore/he.dm

```diff
@@ -1,46 +1,451 @@
+
+	"Many of those who underwent an operation inside the machine have found themselves invigorated and healthy again. Those who answered “Yes” to the question “Do you love your City?” were allowed to become strong. Those who answered “No” had to become strong. The longer the machine is in use, the stronger a person becomes. Furthermore, the longer it is used, the more a person's body would fall in line with the machine's idea of strong."<br>
+
+	<h3><center>Excerpt from Counseling Log</center></h3><br>
+	"Yeah, I'll admit it. It was really stupid of me to try and take a selfie next to the damn machine. I don't know, it just was kinda stationary? I didn't think it'd suddenly turn on when I leaned my elbow on it...<br>
+	But hey, I don't think it was all that bad. I know I was screaming like hell and the pain was unbearable, but once the machine let go, it was just... done. Couldn't feel anything. Now my arm feels lighter, yet more durable.<br>
+	Okay, I didn't know that was how <Redacted>'s skull was going to react to me punching him. He deserved it for grabbing me when I told him to knock it off and all, but I didn't think I'd punch him so... hurtily.<br>
+	Still, even though I don't know my own strength, this arm's pretty cool."<br>
+
+	<h3><center>Urgent Report of the Incident related to You Must Become Strong</center></h3><br>
+	"We’ve filed this urgent report due to the repeated use of You Must Become Strong's limb augmentations. Due to their excessive use causing avoidable damage, as well as unpredicted behaviors from the machine, we are now requiring clearance from the Manager before anyone can receive the Abnormality's augments. If anyone is caught with an unpermitted augmentation from the Abnormality, they will be subject to termination even in the case of negligence or accident."<br>
+
+	<h3><Warning>  Only employees Level 3 and higher are permitted to read this document.</h3><br>
+	"(This recording contains a call from the Containment Unit’s emergency phone. Phone systems are installed in every room of Abnormalities of Risk Level HE or higher to enable quick threat response.)<br>
+	Employee: Breach detected! I'm entering the containment unit right now!... What the?<br>
+	???: Ah. Ah. Highly excited!!! / Ahh~ Thoroughly amusing!!! / Da GAME of DEATH! (Three of the same voice can be heard speaking at different times.)<br>
+	Employee: The hell are those things?<br>
+	???: Strong! STRONG!!! / So excited!!! / Worktime! Worktime!<br>
+	Employee: Uh... Okay... Hello, ERD? This is the Containment Unit of 'You Must Become Strong...?'<br>
+	The Emergency Response Department (ERD): Hello, this is the Emergency Response Team. What’s the problem?<br>
```

Comment:
I'd replace ERD with Safety or welfare, minor thing

Reply:
All set! ERD was a carryover from Nothing There's lore, but I don't think it appears in any other lore log so I can rewrite it for consistency when I get to porting in NT. Thank you!